CMakeLists.txt

```diff
@@ -3,13 +3,26 @@ set(CMAKE_CXX_STANDARD 20)
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
 set(CMAKE_OSX_ARCHITECTURES "x86_64")
 set(CMAKE_CXX_VISIBILITY_PRESET hidden)
+set(CMAKE_BUILD_TYPE "RelWithDebInfo")
```

Comment:
I am fairly sure this does nothing, by the time the cmake is running its already too late to change the build configuration

Reply:
@matcool did the wrong variable, that one's for single config generators. It now uses CMAKE_CONFIGURATION_TYPES so that it'll work with things like xcode and visual studio

Reply:
it is still very windows centric as you can actually build for Debug on other platforms like android and macos

Reply:
> it is still very windows centric as you can actually build for Debug on other platforms like android and macos

@altalk23 it now depends on the platform